tools/workspace-plugin/src/generators/split-library-in-two/generator.ts

```diff
@@ -167,11 +167,11 @@ function makeSrcLibrary(tree: Tree, options: Options, logger: CLIOutput) {
   const newProjectSourceRoot = joinPathFragments(newProjectRoot, 'src');
 
   visitNotIgnoredFiles(tree, oldProjectRoot, file => {
-    if (file.includes('/stories/') || file.includes('/.storybook/')) {
+    if (file.includes(path.normalize('/stories/')) || file.includes(path.normalize('/.storybook/'))) {
```

Comment:
bug: `'/stories/'` was never going to properly resolve on windows shell which uses `\` - fixed